site/index.html

```diff
@@ -135,12 +132,12 @@ <h2>daily runner of <a href="https://github.com/tc39/test262">test262</a> for <i
       sucrase: 'transformers'
     };
 
-    const niceEngineOrder = [ 'v8', 'v8_exp', 'sm', 'sm_exp', 'jsc', 'hermes', 'qjs', 'qjs_ng', 'libjs', 'chakra', 'graaljs', 'xs', 'rhino', 'nashorn', 'boa', 'kiesel', 'porffor', 'nova', 'engine262', 'babel', 'swc', 'sucrase' ];
+    const niceEngineOrder = [ 'v8', 'v8_exp', 'sm', 'sm_exp', 'jsc', 'hermes', 'qjs', 'qjs_ng', 'libjs', 'chakra', 'graaljs', 'xs', 'rhino', 'boa', 'kiesel', 'porffor', 'nova', 'engine262', 'babel', 'swc', 'sucrase' ];
 
     const thead = document.querySelector('thead');
     const tbody = document.querySelector('tbody');
 
-    let cwd = '', filterOutEngines = [ 'v8_exp', 'sm_exp', 'xs', 'rhino', 'nashorn', 'boa', 'qjs_ng', 'engine262', 'kiesel', 'porffor', 'nova', 'swc', 'sucrase' ], init = false;
+    let cwd = '', filterOutEngines = [ 'v8_exp', 'sm_exp', 'xs', 'rhino', 'boa', 'qjs_ng', 'engine262', 'kiesel', 'porffor', 'nova', 'swc', 'sucrase' ], init = false;
```

Comment:
The `filterOutEngines.length !== 13` needs changing below too :sweat_smile:

Reply:
:raised_eyebrow:

Done & left a comment :D